Math/NumberTheory/DirichletCharacters.hs

```diff
@@ -0,0 +1,589 @@
+-- |
+-- Module:      Math.NumberTheory.DirichletCharacters
+-- Copyright:   (c) 2018 Bhavik Mehta
+-- Licence:     MIT
+-- Maintainer:  Bhavik Mehta <bhavikmehta8@gmail.com>
+-- Stability:   Provisional
+-- Portability: Non-portable (GHC extensions)
```

Comment:
I usually ignore `Stability` and `Portability`, IMHO they do not make much sense nowadays.

Reply:
Fair enough, I will remove.